CONTRIBUTING.md

```diff
@@ -0,0 +1,43 @@
+# Contributing to Manticore
+
+First, thanks for your interest in contributing to Manticore! We welcome and
+appreciate all contributions, including bug reports, feature suggestions,
+tutorials/blog posts, and code improvements.
+
+If you're unsure where to start, we recommend our `easy` and `help wanted`
```

Comment:
This is pretty nitpicky, but how would you feel about substituting the code-formatted "easy" and "help wanted" with links to the labels themselves? (e.g. https://github.com/trailofbits/manticore/issues?q=is%3Aissue+is%3Aopen+label%3Aeasy)

Reply:
Awesome idea! Being nitpicky is exactly what i want ;)